section/git-commit-hooks/README.md

```diff
@@ -6,10 +6,18 @@ With [Husky](https://typicode.github.io/husky/#/), you can add any number of com
 
 ## Setting up Husky
 Setting up Husky is incredibly easy as Husky provides a command that will install, initialize, and create all files needed for Husky to start working. There is a manual process if the single set up command doesn't work so if you run into problems, you can try the set up step by step.
+
+### Yarn
 ```
 npx husky-init && yarn
 ```
 
+### NPM
+```
+npx husky-init && npm install
+```
+
+
```

Comment:
I've simple tried to execute instruction step by step without having yarn installed and failed. this fix will include also instructions for NPM